### DISCUSSION TITLE
Is it possible to show which command(bash) is running through airflow Logs ?

### DISCUSSION THREAD

**Discussion by Nidesh-Arumugam** [apache/airflow#21762] [Q&A] [answered]
Well in my case, we will get a list of bash commands as a param from frontend  and pass it to airflow bash operator. 
If we receive "cd / && df -h && ls -la && pwd" as an input it will execute all the commands and in a straight manner without any identification on which command is running.

For example,
If airflow runs df -h
// logs for df -h comes here
Then airflow runs ls -la
// logs for ls -la comes here

What i need is 
If airflow runs df -h
Executing command : df -h              //some sort of identification on which command is executing currently.
// logs for df -h comes here
Then airflow runs ls -la
Executing command : ls -la
// logs for ls -la comes here

Is there any way we can add them, If yes how can i do them.

**Comment by potiuk** [comment, marked as answer]
You can utilze bash's `-x` switch 

You can run this command: `/bin/bash -x -c "cd && ls"`

```
/bin/bash -x -c "cd && ls"
+ cd
+ ls
Applications			Downloads			Movies				Public				getting-started			liquidprompt
Desktop				IdeaProjects			Music				airflow				iCloud Drive (Archive)		tmp_venv
Documents			Library				Pictures			code				java_error_in_idea.hprof
```

Or you can run separate bash script doing it:

```shell
#!/usr/bin/env bash
set -x
cd / && df -h && ls -la && pwd
```

**Reply by Nidesh-Arumugam** [reply]
thx for the reply, we solved it by adding echo in the front.
